script.js

```diff
@@ -34,20 +34,27 @@ const QUOTES = [
   "When you feel the liquid enter my body, I know I'm about to be penetrated",
 ];
 
-const pickQuote = () => {
+function getQuoteText() {
   const queryParams = new URL(document.location).searchParams;
 
   if (queryParams.get('quote') !== null) {
-    return parseInt(queryParams.get('quote'));
+    return QUOTES[parseInt(queryParams.get('quote'))];
   }
 
   const today = new Date();
+
+  if (today.getDate() === 20 && today.getMonth() === 2) {
```

Comment:
unrelated but 0 index'ing things that are literally already numbered is stupid i hate it

Reply:
The best part is date isn't 0 indexed but month is lol

Reply:
Literally the worst I hate it so much